style.css

```diff
@@ -702,10 +821,54 @@ input[type=submit]:hover {
     div.grid_4_headers{
 		height: 160px;
 	}	
+
+	div.slideshow img, div.slideshow{
+	  height: 600px;
+	}
+	
+	div.mySlides{
+	  height: 550px
+	}
+	
+	div.headers_text p{
+	top: 350px;
+	}
+
+	div.headers_text h1{
+	top: 75px;
+	}
+	
+	div.animals{
+	grid-column-start: 2;
+	grid-column-end: 13;
+    }
+	
 	
-	 div.about_us_A img{
-	  width: 45%;
+	div.what_to_bring_tips{
+	grid-column-start: 2;
+	grid-column-end: 14;
     }
+	
+}
+
+/* On screens under 2500px wide and above 1500px, decrease content size */

```

Comment:
This was added as I needed to have a specific height and styling for some elements at this width instead of 1500px to 2500px;